package.json

```diff
@@ -4,29 +4,34 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
-    "@gnosis.pm/safe-apps-sdk": "^0.1.1",
-    "@gnosis.pm/safe-react-components": "^0.1.3",
+    "@gnosis.pm/safe-apps-sdk": "0.4.0",
+    "@gnosis.pm/safe-react-components": "https://github.com/gnosis/safe-react-components.git#be51ca5",
     "@material-ui/core": "^4.9.0",
+    "@rmeissner/safe-apps-react-sdk": "^0.3.4",
```

Comment:
Should functions in this repository be on some @gnosis repository? Is something pending?